js/src/components/google-combo-account-card/account-creation-description.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+const AccountCreationDescription = ( {
+	isCreatingAccounts,
+	isCreatingMCAccount,
+	isCreatingAdsAccount,
+	googleAccount = {},
```

Comment:
@ankitrox `googleAccount` will always have a value since we are showing the connected card once we have a Google account connected. So we can omit the default value here.
```suggestion
	googleAccount,
```

Reply:
@ankitrox Also we are drilling this prop from `js/src/components/google-combo-account-card/index.js` through `js/src/components/google-combo-account-card/connected-google-combo-account-card.js` to this file.
Why not use the `useGoogleAccount` hook here instead to get the connected email address. Because at this point, we'll already have the data in the store.

Reply:
Removed this prop and used `useGoogleAccount` hook. Thanks